README.md

```diff
@@ -49,4 +49,14 @@ The rendered output of the above JSX statement will place the textual content of
 </div>
 ```
 
-For more examples, see the tests written in [src/spec.tsx](https://github.com/ca-la/react-portal/blob/update-metadata/src/spec.tsx).
+For more examples, see the tests written in [src/spec.tsx](https://github.com/ca-la/react-portal/blob/master/src/spec.tsx).
```

Comment:
Maybe use a [relative link?](https://github.blog/2013-01-31-relative-links-in-markup-files/) Makes it valid when you're clicking the link on a branch's README.

Reply:
Ooh yeah that's exactly what I'm looking for!

Reply:
6f474f0